_helpers/context_aware_executor.py

```diff
@@ -0,0 +1,34 @@
+from concurrent.futures import ThreadPoolExecutor
+from contextvars import copy_context
+
+
+class ContextAwareExecutor:
```

Comment:
Do we need to redefine this here as it's already in notification?
app/notification/scheduler/context_aware_executor.py
https://github.com/communitiesuk/funding-service-design-notification/pull/233/files#diff-f0964165f72c7f2ce4fd79324539d9c286a02df206da7f35498ad1fbf2433f96R5

Reply:
Fixed